features/gezag-minderjarige/geen-gezag.feature

```diff
@@ -0,0 +1,29 @@
+#language: nl
+
+Functionaliteit: Over meerderjarige of overleden personen wordt geen gezag geleverd
+  Gezag over een minderjarige stopt als het kind 18 jaar wordt of is overleden. 
+
+    Achtergrond:
+    Gegeven de persoon 'Bert' met burgerservicenummer '000000036'
+      * is meerderjarig
+      * is in Nederland geboren
+      * is ingeschreven in de BRP
+      Dan heeft de response een persoon met de volgende gegevens
+        | naam                | waarde    |
+        | burgerservicenummer | 000000036 |
+
+
+  Regel: Als de persoon meerderjarig is, dan is er geen sprake van gezag
+
+    Scenario: De persoon is meerderjarig
```

Comment:
Volgens mij wordt hier impliciet gezegd dat de meerderjarige geen minderjarige kinderen heeft.

NB. Volgens mij heb ik hier een mogelijke definitie voor een kind. Een persoon is een kind als hij bij één of twee personen staat geregistreerd (in de BRP) als kind.